app/containers/Header/Notification.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import Dropdown from 'components/Dropdown';
+import Li from 'components/Li';
+import H4 from 'components/H4';
+import AddText from 'components/AddText';
+
+const ButtonStyled = styled.span`
```

Comment:
We will not reuse that styles anywhere else? does not need to be a separate component?

Reply:
it isn't reused now. I'll create separate file If it will be needed